### PR TITLE
updated slo summary widget initial updates

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by roxanne-moslehi** [comment]
Updates to make:
- update screenshot used

File: content/en/dashboards/widgets/slo.md [hunk 1]
```diff
@@ -21,43 +21,48 @@ further_reading:
...
+
+- For SLOs with groups (metric-based or Time Slice SLO with groups, or monitor-based SLOs with a single monitor broken into groups), there are the following three view modes:
   - `Overall`: displays the overall SLO status percentages and targets
   - `Groups`: displays a table of status percentages for each group
   - `Both`: displays both the overall SLO status percentages and targets and table of status percentages for each group
 
 - For monitor-based SLOs configured with multiple monitors, there are the following three view modes:
   - `Overall`: displays the overall SLO status percentages and targets
-  - `Groups`: displays a table of status percentages for each group
+  - `Groups`: displays a table of status percentages for each monitor
```
Comment (mmeyer724) [inline content/en/dashboards/widgets/slo.md:58]:
```suggestion
  - `Monitors`: displays a table of status percentages for each monitor
```
Looks like an error in our existing documentation. For multi-monitor SLOs, this option is named 'Monitors'

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, requested some minor edits and clarifications.

Additionally, it looks like the new screenshot shows information about Datadog data centers. Could you re-load the image with obfuscated group values, or otherwise provide a fresh image?

**Comment by roxanne-moslehi** [comment]
Thanks @buraizu for the comments - I will make updates. We will still need to wait to merge this PR since we are testing out the new features still. For the image, I just have that as a placeholder and will update, but the groups will always be cut off since it's a scrollable preview of the groups.

**Comment by roxanne-moslehi** [comment]
@buraizu also FYI @estherk15 - We are going to start releasing this feature next week and will need to merge in the updated docs. I've made all changes on my end and will follow up when I'm ready to merge this in?

**Comment by estherk15** [comment]
@roxanne-moslehi There's one unresolved suggestion: https://github.com/DataDog/documentation/pull/21858/files#r1496299959
After Bryce approves, this is good to go, feel free to ping either of us or #documentation to merge!

**Comment by roxanne-moslehi** [comment]
@estherk15 @buraizu This is ready to merge now

**Review by estherk15 — APPROVED** [review]

**Pull request merged**